classes/options-general.php

```diff
@@ -892,7 +892,7 @@ class="radio-thumbnail<?php if ( 'TERRAIN' == $map_type ) {
 
 											<p class="submit" align="right">
 												<input type="submit" class="button-primary"
-												       value="<?php _e( 'Save Options', 'simplemap' ) ?>"/>&nbsp;&nbsp;
+												       value="<?php _e( 'Save Options', 'SimpleMap' ) ?>"/>&nbsp;&nbsp;
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->